crates/uv-trampoline/src/bounce.rs

```diff
@@ -275,7 +275,26 @@ fn find_python_exe(executable_name: &CStr) -> CString {
         String::from("Failed to close file handle")
     });
 
-    path
+    if !path.as_bytes().starts_with(b"..") {
```

Comment:
Is there no `has_root` or similar that we can use here? There are lots of relative paths that won't necessarily start with this.

Reply:
I've never done work without the standard library so I'm not sure what is possible.